responsive-test-utils/src/main/java/dev/responsive/kafka/api/ResponsiveTopologyTestDriver.java

```diff
@@ -16,31 +16,23 @@
 
 package dev.responsive.kafka.api;
 
+import dev.responsive.kafka.clients.TTDCassandraClient;
+import dev.responsive.kafka.clients.TTDMockAdmin;
 import dev.responsive.kafka.config.ResponsiveConfig;
-import dev.responsive.kafka.store.CassandraClientStub;
 import java.time.Duration;
 import java.time.Instant;
-import java.util.Collection;
-import java.util.Collections;
 import java.util.Properties;
 import java.util.concurrent.ScheduledThreadPoolExecutor;
-import org.apache.kafka.clients.admin.DescribeTopicsResult;
-import org.apache.kafka.clients.admin.MockAdminClient;
-import org.apache.kafka.common.Node;
-import org.apache.kafka.common.TopicPartitionInfo;
-import org.apache.kafka.common.config.TopicConfig;
-import org.apache.kafka.common.serialization.Serializer;
 import org.apache.kafka.common.utils.MockTime;
 import org.apache.kafka.streams.StreamsConfig;
-import org.apache.kafka.streams.TestInputTopic;
 import org.apache.kafka.streams.Topology;
 import org.apache.kafka.streams.TopologyDescription;
 import org.apache.kafka.streams.TopologyTestDriver;
-import org.apache.kafka.streams.test.TestRecord;
 
 public class ResponsiveTopologyTestDriver extends TopologyTestDriver {
+  public static final String RESPONSIVE_TTD_ID = "Responsive_TopologyTestDriver";
```

Comment:
will there be any issues if this is reused? would it make sense to add a random id to it like the TTD does?

Reply:
Nah, I can't imagine how there could be -- there's no real "cluster", everything is internal to the TTD. Even the StateDirectory is wiped when the TTD is closed

I guess if you have really poor test practices and don't ensure the TTD is closed, have a persistent/rocksdb store somewhere in your topology alongside the Responsive stores, AND set the state directory yet not the app id (because default state.dir involves randomly generated name IIRC), then this could be a "regression" -- but at that point, no one can help you lol